### DISCUSSION TITLE
Mail from quarkus community

### DISCUSSION THREAD

**Discussion by luca-bassoricci** [quarkusio/quarkus#30232] [Community]
I have received this mail. 
Is this a real email or phising? 
Thanks 

Thank You For Making Quarkus 3.0
--
Hey Luca Basso Ricci
We would like to extend a big thank you for making the Quarkus 3.0 release such a huge success with a custom "I Made Quarkus 3.0" t-shirt. You were selected based on your GitHub contribution(s) since the 2.1 release.Please fill out this form with your information and we will ship you a shirt.

**Comment by maxandersen** [comment]
It is legit. Not a scam.

**Reply by kdubois** [reply]
@maxandersen will there be another round of emails going out for people who got pr's approved after that first email went out?   Asking for a friend :smile:

**Reply by luca-bassoricci** [reply]
> It is legit. Not a scam.

I've received my tshirt today 👍

**Reply by maxandersen** [reply]
@kdubois there will be a follow up. 

Right @insectengine ? :)

**Reply by insectengine** [reply]
absolutely!

**Reply by igorregis** [reply]
I have received this mail also and I remember to fill in the form, but I've got no shirt. I've 

Any restriction to send it abroad? I live in Brazil.

**Reply by insectengine** [reply]
Hello @igorregis,

I'm sorry about the confusion. I've checked the form submittal responses but I unfortunately can't find your name. There must have been an error. Please email me at jcobb@redhat.com and we'll see if we can work something out.

Regards, James

**Reply by igorregis** [reply]
Thanks for the reply @insectengine!
